validator/src/main.rs

```diff
@@ -2117,7 +2117,10 @@ pub fn main() {
             _ => unreachable!(),
         };
 
-    let mut accounts_index_config = AccountsIndexConfig::default();
+    let mut accounts_index_config = AccountsIndexConfig {
+        started_from_validator: true, // this is the only place this is set
+        ..AccountsIndexConfig::default()
+    };
```

Comment:
Should we add similar in `bin/solana-test-validator.rs`?  It's a dapp dev tool, not intended for core testing.

Reply:
plan to do this in a separate pr. Looks like some plumbing is required.